docs/fundamentals/runtime-libraries/system-environment-getenvironmentvariable.md

```diff
@@ -9,14 +9,17 @@ ms.date: 01/24/2024
 
 The <xref:System.Environment.GetEnvironmentVariable%2A> method retrieves the value of an environment variable from the current process.
 
+Environment variable names are case-sensitive on Unix-like systems but aren't case-sensitive on Windows.
+
+> [!NOTE]
+> In-process environment modifications made by native libraries aren't seen by managed callers.
```

Comment:
Great, thanks for tackling this too. It  may make sense to cover _both_ directions here, by adding something like: "Conversely, such modifications made by managed callers aren't seen by native libraries."